### PR TITLE
Add documentation for Supervisor Update entities

### PR DESCRIPTION
**Pull request by ludeeus**
## Proposed change

Adds documentation for Supervisor update entities

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase:  https://github.com/home-assistant/core/pull/68475
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/hassio.markdown [hunk 1]
```diff
@@ -51,6 +53,10 @@ For Home Assistant OS Supervisor provides following binary sensors:
 | ------- | ------------------ | ----------- |
 | Update Available | no | Whether there is an update available for OS
 
+## Update entities
+
+For all your installed add-ons, Home Assistant Core, Home Assistant Supervisor and for the Home Assistant Operating System (if you are running that), this integration will provide [update](/integrations/update) entities that provide information about bending updates, and will allow you to update to them.
```
Comment (frenck) [inline source/_integrations/hassio.markdown]:
```suggestion
For all your installed add-ons, Home Assistant Core, Home Assistant Supervisor, and for the Home Assistant Operating System (if you are running that), this integration will provide [update](/integrations/update) entities that provide information about pending updates, and will allow you to update to them.
```

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

File: source/_integrations/hassio.markdown (hunk 1 above)
Comment (ludeeus) [reply source/_integrations/hassio.markdown]: So we are not bending updates? 😂

**Pull request merged**